pinot-controller/src/test/java/org/apache/pinot/controller/api/TableViewsTest.java

```diff
@@ -161,6 +167,55 @@ public void testHybridTableState(String state)
     assertEquals(tableView._realtime.size(), DEFAULT_NUM_SERVER_INSTANCES);
   }
 
+  @Test
+  public void testJsonDeserializationSegmentStatusInfo()
```

Comment:
Suggest moving them into a separate test `SegmentStatusInfoTest`

Reply:
Moving it to SegmentStatusCheckerTest.